bindings/nodejs/lib/types/block/output/feature.ts

```diff
@@ -11,82 +13,72 @@ enum FeatureType {
     Issuer = 1,
     Metadata = 2,
     Tag = 3,
+    BlockIssuer = 4,
 }
-
 abstract class Feature {
-    private type: FeatureType;
+    readonly type: FeatureType;
     constructor(type: FeatureType) {
         this.type = type;
     }
-    /**
-     * The type of feature.
-     */
-    getType(): FeatureType {
-        return this.type;
-    }
 }
+
 /**
  * Sender feature.
  */
 class SenderFeature extends Feature {
-    private address: Address;
+    readonly address: Address;
     constructor(sender: Address) {
         super(FeatureType.Sender);
         this.address = sender;
     }
-    /**
-     * The address.
-     */
-    getSender(): Address {
-        return this.address;
-    }
 }
 /**
  * Issuer feature.
  */
 class IssuerFeature extends Feature {
-    private address: Address;
+    readonly address: Address;
     constructor(issuer: Address) {
         super(FeatureType.Issuer);
         this.address = issuer;
     }
-    /**
-     * The address.
-     */
-    getIssuer(): Address {
-        return this.address;
-    }
 }
 /**
  * Metadata feature.
  */
 class MetadataFeature extends Feature {
-    private data: string;
+    readonly data: string;
     constructor(data: string) {
         super(FeatureType.Metadata);
         this.data = data;
     }
-    /**
-     * Defines metadata (arbitrary binary data) that will be stored in the output.
```

Comment:
Can this comment be moved to the field or does this make no sense, because it won't show up in an IDE?

Reply:
I think the IDE will show documentation on fields, I added it to the field.